katalog/prometheus-operated/rules.yml

```diff
@@ -1366,7 +1366,7 @@ spec:
       for: 10m
       labels:
         severity: warning
-    - alert: Watchdog
+    - alert: DeadMansSwitch
```

Comment:
This is for convenience sake - to avoid changing alertmanager configs.
It removes upstream compatibility.